docs/source/references/config.md

```diff
@@ -595,72 +588,6 @@ Has type `u64`. Can be configured via environment variable `SUMERAGI_TX_RECEIPT_
 500
 ```
 
-## `telemetry`
```

Comment:
I think documentations should cover all the features

Reply:
Iroha's feature flags need more attention than they received.